saleor/graphql/account/mutations.py

```diff
@@ -130,3 +173,23 @@ def clean_input(cls, info, instance, input, errors):
     def save(cls, info, instance, cleaned_input):
         instance.set_password(cleaned_input['password'])
         instance.save()
+
+
+class PasswordReset(BaseMutation):
+    class Arguments:
+        email = graphene.String(description='Email', required=True)
+
+    class Meta:
+        description = 'Sends password reset email'
+
+    @classmethod
+    @permission_required('account.manage_users')
+    def mutate(cls, root, info, email):
+        try:
+            user = models.User.objects.get(email=email)
+        except ObjectDoesNotExist:
+            return cls(
+                errors=[
+                    Error(field='email',
+                          message="User with this email doesn't exist")])
```

Comment:
To be consistent with formatting in rest of the file, please use single quotes here.

Reply:
We use a bit different line-breaking style - `field` should go under Error and `message` under `field`, like this:
```
Error(
    field='email',
    message="User with this email doesn't exist")
```
You may try using `yapf` for code formatting, it should handle it for you.